auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -335,11 +390,79 @@ export default defineComponent({
       clearAllFilters()
     }
 
+    const isCurrentOrganization = (orgId: number) => {
+      return orgId === store.state.org.currentOrganization.id
+    }
+
+    const actionHandler = (business: Business) => {
+      const aii = business.affiliationInvites[0]
+      const status1 = aii.status
+      if (status1 === AffiliationInvitationStatus.Pending) {
+        OrgService.cancelAffiliationInvitation(aii.fromOrg.id, aii.id)
+      } else if (status1 === AffiliationInvitationStatus.Failed) {
+        // remove from list action
+        OrgService.removeAffiliationInvitation(aii.fromOrg.id, aii.id)
+      } else if (status1 === AffiliationInvitationStatus.Accepted) {
+        open(business)
+      } else {
+        return () => {
+        } // do nothing
```

Comment:
This might seem like a dumb question, but why even return?  The other functions aren't returning

Reply:
true, removed the return